code/game/objects/items/storage/firstaid.dm

```diff
@@ -339,12 +340,30 @@
 	. = ..()
 	update_icon()
 
+/obj/item/storage/pill_bottle/refill(mob/user)
```

Comment:
Pill bottles are currently neither infinite nor restockable, this is beyond qol.

Reply:
Pill packets are infintie.
I've added the flag specifically to pill bottles that have a packet variant